spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -316,17 +387,33 @@ function onTimeoutPacket(packet: Packet) {
 
 ```typescript
 function refundTokens(packet: Packet) {
-  FungibleTokenPacketData data = packet.data
+  channel = provableStore.get(channelPath(portIdentifier, channelIdentifier))
```

Comment:
Should we also do here `transferVersion = getAppVersion(channel.version)`?

Reply:
it's done below